src/bootstrap/cloud/terraform/service-account.tf

```diff
@@ -71,8 +71,7 @@ resource "google_project_iam_member" "robot-service-roles" {
 resource "google_project_iam_member" "robot-service-kubernetes" {
   project = data.google_project.project.project_id
 
-  # TODO(rodrigoq): migrate to RBAC and remove roles/container.developer
-  role = var.cr_syncer_rbac == "true" ? "roles/container.clusterViewer" : "roles/container.developer"
+  role = "roles/container.clusterViewer"
 
   member = "serviceAccount:${google_service_account.robot-service.email}"
 }
```

Comment:
can you fold this into the robot-service-roles?

Reply:
Done, I'm worried that Terraform won't cope with the rename so I've added a line to deploy.sh to make sure it's maintained.